src/Forte.Optimizely.ContentUsage/Frontend/Views/ContentTypesView.tsx

```diff
@@ -185,93 +168,15 @@ const ContentTypesView = () => {
-                        onRowClick={onTableRowClick(
-                          row.guid,
-                          row.displayName || row.name
-                        )}
-                      >
-                        {tableColumns
-                          .filter((column) => column.visible)
-                          .map((column) => (
-                            <Table.TD key={column.id}>
-                              {row[column.id]}
-                            </Table.TD>
-                          ))}
-                        <Table.TD verticalAlign="middle">
-                          <Dropdown
-                            activator={
-                              <ButtonIcon
-                                iconName="ellipsis"
-                                size="small"
-                                style="plain"
-                                title={actions.title}
-                              />
-                            }
-                          >
-                            <Dropdown.Contents>
-                              <Dropdown.ListItem
-                                onClick={onTableRowClick(
-                                  row.guid,
-                                  row.displayName || row.name,
-                                  true
-                                )}
-                              >
-                                <Dropdown.BlockLink>
-                                  <Dropdown.BlockLinkText
-                                    text={actions.viewUsages}
-                                  />
-                                </Dropdown.BlockLink>
-                              </Dropdown.ListItem>
-                              <Dropdown.ListItem>
-                                <CopyToClipboard text={row.guid}>
-                                  <Dropdown.BlockLink>
-                                    <Dropdown.BlockLinkText
-                                      text={actions.copyGuid}
-                                    />
-                                  </Dropdown.BlockLink>
-                                </CopyToClipboard>
-                              </Dropdown.ListItem>
-                            </Dropdown.Contents>
-                          </Dropdown>
-                        </Table.TD>
-                      </Table.TR>
-                    ))
-                  ) : (
-                    <Table.TR noHover>
-                      <Table.TD colSpan={6}>{translations.noResults}</Table.TD>
-                    </Table.TR>
-                  )}
-                </Table.TBody>
-              </Table>
+              <ForteTable
+                tableColumns={tableColumns}
+                rows={rows}
+                rowTemplate={(row, columns) => (
+                  <ContentTypeTableRow row={row} tableColumns={columns} />
+                )}
+                sortDirection={sortDirection}
+                onSortChange={onSortChange}
+              />
```

Comment:
I am wondering if instead of passing props we shouldn't rely more on composition, for example `ContentTypeTableRow` could look like this:

```
<TableRow>
  {columns.map(column => (
    <Column ... /> // loop through the array of columns
  )}
  <Column ... /> // additional column you wanted to add at the end
</TableRow>
```

This makes it easier for future customization without adding more and more props, maybe we will need to add an additional column at the start in the future? We can discuss it later if you want to.